mypyc/doc/differences_from_python.rst

```diff
@@ -262,12 +262,6 @@ Descriptors
 Native classes can't contain arbitrary descriptors. Properties, static
 methods and class methods are supported.
 
-Defining protocols
-******************
-
-Protocols can't be defined in compiled code. You can use protocols
-defined elsewhere, however.
-
```

Comment:
Does this mean protocols can be used now?

Reply:
Yeah, they should work now. Operations on them will fall back to generic operations, though.